tutorials/others/ubuntu/tutorial.yml

```
@@ -0,0 +1,26 @@
+id: 78a3be56-5d51-4ec3-8629-0dd27c352ab5
+
+builder: Canonical
+
+tags:
+  - privacy
+  - open-source
```

```suggestion
  - guides
```
(Tag "open-source" is not in the Paolo's list)

---

Sorry, I've put the same as the Linux mint one instead of checking the Paolo's list ^^ Maybe we'll need to change this one too ?

---

Yes, indeed. We implemented the tag list a few months ago, but all tutorials written before that don’t follow this logic. You’re right I’ll update them 👍